interfaces/builtin/bool_file_test.go

```diff
@@ -174,12 +174,8 @@ func (s *BoolFileInterfaceSuite) TestConnectedPlugSnippetPanicksOnUnsanitizedSlo
 
 func (s *BoolFileInterfaceSuite) TestConnectedPlugSnippetUnusedSecuritySystems(c *C) {
 	for _, slot := range []*interfaces.Slot{s.ledSlot, s.gpioSlot} {
-		// No extra seccomp permissions for plug
-		snippet, err := s.iface.ConnectedPlugSnippet(s.plug, slot, interfaces.SecuritySecComp)
```

Comment:
Are we loosing a test here (and below)? I.e. should we do something like: 
```
// No extra seccomp permissions for plug
seccompSpec := &seccomp.Specification{}
err := seccompSpec.AddConnectedPlug(s.iface, s.plug, slot)
c.Assert(err, IsNil)
sn := seccompSpec.Snippets()
c.Assert(sn, HasLen, 0)
```
here (and below)?